src/dispatch/participant/flows.py

```diff
@@ -13,58 +15,75 @@
 
 log = logging.getLogger(__name__)
 
+Obj = TypeVar("Obj", Case, Incident)
+
 
 def add_participant(
     user_email: str,
-    incident: Incident,
+    obj: Obj,
```

Comment:
When a function could be either an `Incident` or a `Case`. I have used `subject` as the variable name as it's a bit more descriptive.

Reply:
I used `obj` but I don't have strong opinions between one vs other as long as we're consistent across the codebase.

Reply:
I'll standardize on `subject` to keep it consistent.